detector.js

```diff
@@ -62,23 +62,27 @@ export default class Detector {
     constructor(containerElement, params) {
         params = Object.assign({}, defaultParams, params);
 
-        const canvasWrapper = document.createElement('div');
+        // const canvasWrapper = document.createElement('div');
+        // canvasWrapper.className = 'detector-canvas-wrapper';
+        // const canvasWrapper = document.getElementsByClassName('detector-canvas-wrapper');
+        const canvasWrapper = document.getElementById('face-mesh-video');
         canvasWrapper.className = 'detector-canvas-wrapper';
-
-        const canvas = document.createElement('canvas');
+        console.log(canvasWrapper)
+        // const canvas = document.createElement('canvas');
+        const canvas = document.getElementById('output');
         canvas.className = 'detector-overlay';
 
-        const video = document.createElement('video');
-        video.setAttribute('playinline', '');
-        video.setAttribute('muted', '');
-        video.style = `transform: scaleX(-1);
-            display: none;
-            width: auto;
-            height: auto;`;
+        // const video = document.createElement('video');
+        const video = document.getElementById('video');
+        // video.setAttribute('playinline', 'playinline');
+        // video.style = `transform: scaleX(-1);
+        //     visibility: hidden;
+        //     width: auto;
+        //     height: auto;`;
 
         canvasWrapper.appendChild(canvas);
         canvasWrapper.appendChild(video);
-        containerElement.appendChild(canvasWrapper);
+        // containerElement.appendChild(canvasWrapper);
```

Comment:
We need to work this so we don't have to modify the creation of DOM inside of this class. Ideally, I think everything should create by creating this object without hooking on the existing HTML element